app/assets/stylesheets/elements/_codemirror.scss

```diff
@@ -114,7 +114,9 @@
   .cm-completionIcon {
     padding-right: 1rem;
   }
-
+  .cm-completionIcon-event::after {
+    content: "∈";
+  }
```

Comment:
What's this for?

Reply:
its the symbol shown in the editor, E for Event!
![image](https://user-images.githubusercontent.com/85789932/229614207-44d4ad46-9e7e-4171-ae1a-08e401ea1586.png)